src/hardware/mouse/mouse_config.cpp

```diff
@@ -39,9 +38,16 @@ CHECK_NARROWING();
 
 // #define ENABLE_EXPLORER_MOUSE
 
-MouseConfig mouse_config;
+MouseConfig     mouse_config;
 MousePredefined mouse_predefined;
 
+static struct {
+	const std::string param_seamless = "seamless";
+	const std::string param_onclick  = "onclick";
+	const std::string param_onstart  = "onstart";
+	const std::string param_nomouse  = "nomouse";
+} capture_types;
+
```

Comment:
OK, got rid of structures.

Alternative approach might be to rework the configuration API, so that it would accept list of pairs like <enum, string>. Would require a template... but this is definitely out of scope of this PR.